libr/anal/esil.c

```diff
@@ -356,6 +361,7 @@ static int internal_esil_mem_write(RAnalEsil *esil, ut64 addr, const ut8 *buf, i
 
 static int internal_esil_mem_write_no_null(RAnalEsil *esil, ut64 addr, const ut8 *buf, int len) {
 	int ret = 0;
+	addr = addr & esil->addrmask;
```

Comment:
use &= instead of addr = addr & ..

Reply:
Yep indeed, that was moronic to do that :D